src/openeo_test_suite/tests/general/payload_examples/patch_jobs/patch_jobs_1.json

```diff
@@ -0,0 +1,342 @@
+{
+    "title": "NDVI based on Sentinel 2",
+    "description": "Deriving minimum NDVI measurements over pixel time series of Sentinel 2",
+    "process": {
```

Comment:
The process object should only contain properties that are actually needed. enums with null values, min/maxItems set to 0 doesn't make sense, etc.

Reply:
I've replaced it with the given example from the 1.2.0 spec